src/Logging.jl

```diff
@@ -41,7 +36,19 @@ catch
     Base.isless(x::LogFacility, y::LogFacility) = isless(x.val, y.val)
 end
 
-type SysLog
+abstract LogOutput
+
+const default_log_format = "%(timestamp):%(level):%(loggername):%(msg)\n"
```

Comment:
Can we tag on the newline somewhere else? It seems weird to have the newline in the format.